docs/tools/libzbc.md

```diff
@@ -31,35 +31,34 @@ the <a href="https://github.com/westerndigitalcorporation/"
 target="_blank">Western Digital Corporation organization on GitHub</a>.
 :::
 
-*libzbc* also provides a test suite allowing to test the conformance of disks
-and HBAs to the ZBC and ZAC standards. In order to make the test suite available,
-*libzbc* needs to be configured using *--with-test* option prior to building and
-installing the library. The usage of this test suite is described in more
-details [here](../tests/zbc-tests.md).
+*libzbc* provides a test suite that makes it possible to test disks and HBAs
+for conformity to the ZBC and ZAC standards.  In order to make the test suite
+available, *libzbc* must be configured using the *--with-test* option prior to
+building and installing the library. The usage of this test suite is described
+in more detail [here](../tests/zbc-tests.md).
 
 ## Overview
 
 *libzbc* provides a unified application programming interface (API) that is
-independent of the zone model and interface of the disk being used. Internally,
-four different types of device drivers are used to handle device interface
-dependent commands.
+independent of the zone model and the interface of the disk being used.
+Internally, four different types of device drivers are used to handle commands that are dependent on the device interface:
 
-* **ZAC ATA Driver** This driver is used to handle direct delivery of ATA
+* **ZAC ATA Driver** This driver is used to handle the direct delivery of ATA
   commands to ZAC disks through the SCSI generic driver (direct device
   access interface).
 * **ZBC SCSI Driver** This driver primarily handles SCSI commands directed at
   ZBC SCSI disks, but can also be used to control ZAC ATA disks if a functional
   SCSI to ATA (SAT) command translation layer is functional in the command path.
```

Comment:
Repeating functional here. We could improve this with something like:
...if a functional SCSI to ATA (SAT) command translation layer exists in the command path.